dali/dfu/dfuwu.cpp

```diff
@@ -2095,6 +2095,16 @@ class CDFUoptions: public CLinkedDFUWUchild, implements IDFUoptions
     {
         queryRoot()->setPropBool("@noCommon",val);
     }
+
+    bool getKeepSourceEncoding() const
+	{
+		return queryRoot()->getPropBool("@keepSourceEncoding", true);
+	}
```

Comment:
@AttilaVamos - there still appears to be something wrong with the formatting/indentation.